src/App.js

```diff
@@ -1,12 +1,67 @@
-import React from 'react';
+import React, {useEffect, useState} from 'react';
 import './App.css';
+import axios from "axios";
+import Pokemon from "./components/pokemon/Pokemon";
+import logo from "./assets/International_Pokemon_logo.png";
+
 
 function App() {
-  return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
-  );
+    const [loading, setLoading] = useState(false)
+    const [error, setError] = useState(false);
+    const [pokemon, setPokemon] = useState([]);
```

Comment:
Als `pokemon` een object zal worden, dan zou ik een default waarde van `undefined` pakken. Nu lijkt het alsof het een array is en zal blijven, maar dat is niet zo
```suggestion
    const [pokemon, setPokemon] = useState();
```